data/aravindraveendran7.json

```diff
@@ -2,8 +2,14 @@
   "name": "Aravind Raveendran",
   "displayStatsPublic": true,
   "type": "personal",
-  "bio": "Test Automation Engineer **Devops Enthusiast**. Stock market enthusiast|Swing Trader|Learning Derviative Trading",
-  "avatar": "https://github.com/aravindraveendran7",
+  "bio": "Test Automation Engineer |DevOps Enthusiast|Stock market enthusiast|Swing Trader|Learning Derviative Trading",
+  "tags": ["Open Source", "DevOps", "Selenium", "Java", "Rest API","Docker","QA Automation"],
+  "avatar": "https://github.com/aravindraveendran7.png",
+  "socials": [
+    { "icon": "FaTwitter",  "url": "https://twitter.com/AravindRaveend6" },
+    { "icon": "FaGithub",   "url": "https://www.linkedin.com/in/aravind-raveendran700" },
+    { "icon": "FaLinkedin", "url": "https://pradumnasaraf.dev" }
```

Comment:
You might want to update this link here.

Reply:
@Panquesito7  sorry for the typo there and thank you for pointing it out 👍 . Raised a new PR now.

Reply:
Please close this PR if you no longer want it

Reply:
Done @eddiejaoude